platforms/jvm/ear/src/main/java/org/gradle/plugins/ear/Ear.java

```diff
@@ -260,7 +260,7 @@ public Property<Boolean> getGenerateDeploymentDescriptor() {
      * The deployment descriptor configuration.
      */
     @Internal
-    @ToBeReplacedByLazyProperty
+    @NotToBeReplacedByLazyProperty(because = "Nested property")
```

Comment:
:x: This is not a nested property, so let's not give that as the reason here.

Reply:
Yes this property is not annotated with `@Nested` but It looks like a nested property just annotated as `@Internal`, I guess so we don't snapshot its values.

Should we migrate this one or what do you think would be a reason here?

Reply:
I changed it to `Nested like property`, not sure what we should put otherwise

Reply:
Optionally we can also try to annotate it as `@Nested` and annotate all properties as `@Internal` if that is any better